<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>686</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QMdiArea" name="mdiArea">
    <property name="geometry">
     <rect>
      <x>-20</x>
      <y>60</y>
      <width>811</width>
      <height>441</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="menufile_F">
    <property name="title">
     <string>file(&amp;F)</string>
    </property>
    <addaction name="NewAction"/>
    <addaction name="showDockAct"/>
   </widget>
   <widget class="QMenu" name="menuEdit_E">
    <property name="title">
     <string>Edit(&amp;E)</string>
    </property>
    <addaction name="openACtion"/>
   </widget>
   <addaction name="menufile_F"/>
   <addaction name="menuEdit_E"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="dockWidget">
   <property name="windowTitle">
    <string>ToolBox</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>4</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <widget class="QPushButton" name="DockerButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Docker</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEdit">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>30</y>
       <width>121</width>
       <height>81</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="NewAction">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/res/new.png</normaloff>:/res/new.png</iconset>
   </property>
   <property name="text">
    <string>new(&amp;N)</string>
   </property>
  </action>
  <action name="showDockAct">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/res/new.png</normaloff>:/res/new.png</iconset>
   </property>
   <property name="text">
    <string>showDock(&amp;D)</string>
   </property>
  </action>
  <action name="openACtion">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/mainwindow/res/new2.png</normaloff>:/mainwindow/res/new2.png</iconset>
   </property>
   <property name="text">
    <string>open(&amp;O)</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
